### DISCUSSION TITLE
Tasmota not react on floor/room any more after firmware update on MQTT

### DISCUSSION THREAD

**Discussion by gwietze** [arendst/Tasmota#19107] [Q&A] [answered]
ON Mqtt at flat                then Topic : domoticz/out  then tasmota is react with domoticz/in  on version 13  firmware
On Mqtt at Floor/Room then Topic: domoticz/out/bg/Kantoor does not react at all on version 13 firmware

When I go back on firmware 11 then Floor/room is working.
Is there a setting in tasmota to get it back to work on version 13?

**Comment by colbrac** [comment]
I have the same experience. I have a couple of Sonoff S26's and added a few more recently. The first set has version 9.2.0 that works with Domoticz/out set to Floor/Room in the Hardware settings for the MQTT Client Gateway with LAN interface device. The recent S26's were flashed with version 13.1.0 and that did not work. 

I have now been downgrading these new S26's and found that:
12.5.0 does not work
12.2.0 does not work
11.1.0 works
10.1.0 works

**Reply by sfromis** [reply]
With senescent devices, the simplest plan may simply be to not upgrade Tasmota to newer versions. For most use cases, the improvements are not of significance. If it ain't broke don't fix it. Older devices may already be borderline in ability to get power from the degraded hardware.

**Reply by colbrac** [reply]
True, but it appears the changes between 11.1.0 and 12.0.0 in [tasmota_xdrv_driver/xdrv_07_domoticz.ino](https://github.com/arendst/Tasmota/compare/d157b1c..7bc7be5?diff=split&w=#diff-3f7d8ab0ef2d415a7e37b27248546a94852f24143a14a265c79b29702b04f207) breaks the reading of subtopics in domoticz/out. A massive change was made to the file, so it is not easy for a novice like me to see what change might cause the degradation in functionality.

**Reply by sfromis** [reply]
The change log is where to see changes and warnings, not the source code.

**Comment by Jason2866** [comment]
ping @arendst since you use Domoticz

**Reply by colbrac** [reply, marked as answer]
I made [an Issue](https://github.com/arendst/Tasmota/issues/20299) and he is working on it.
